org.mixedrealitytoolkit.uxcore/Button/PressableButton.cs

```diff
@@ -182,9 +182,9 @@ public SpaceMode DistanceSpaceMode
         private float WorldToLocalScale => transform.InverseTransformVector(transform.forward).magnitude;
 
         /// <summary>
-        /// Holds the duples Collider + Interactor that have triggered proximity.
+        /// Holds the Interactables that trigger proximity.
         /// </summary>
-        private HashSet<(Collider, XRBaseInteractor)> activeCollidersWithInteractor = new HashSet<(Collider, XRBaseInteractor)>();
+        private HashSet<XRBaseInteractable> activeInteractables = new HashSet<XRBaseInteractable>();
```

Comment:
How would other Interactables cause proximity with this Interactable?  It doesn't seem to make sense that PressableButton (an Interactable) would keep a set of other Interactables in proximity.  It made sense before to keep a set of colliders and associated Interactors that are in proximity.  What is this change addressing?

Reply:
Thank you for contributing to the conversation.  I got confused other feedback and also was under the wrong impression that an interactable would be enough to uniquely identify the source of the interaction and hence proposed this change.  I reverted to use interactors in https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/659/commits/21f16fc4597f04b28bbe910fcfd1c1bc6a21adf0 resolvin this.